### DISCUSSION TITLE
How to change timezone

### DISCUSSION THREAD

**Discussion by cmonty14** [arendst/Tasmota#22281] [Q&A] [answered]
Hi,
according to MQTT data, setting "Timezone" is incorrect in my device:
![image](https://github.com/user-attachments/assets/6c8d9ee6-dab3-4c2f-9880-7083a9d783bc)

Can you please advise how to modify "Timezone" on _Tasmota_?

THX

**Comment by sfromis** [comment, marked as answer]
Recommended tool to generate the commands to get the combined timezone-relevant settings right:
https://tasmotatimezone.com/

I prefer to enable `SetOption52` to have specific time zone included in the time stamps, making the times parseable no matter if the time zone was not quite right.
